everest

```diff
@@ -894,6 +900,24 @@ do_reset ()
   get_vale
 }
 
+do_advance ()
+{
+  for r in $ALL_PROJECTS; do
+    echo
+    blue "Advancing $r"
+    check_subp_exists "$r"
+
+    pushd $r
+    branch=${branches[$r]}
+
+    git checkout -f $branch
+    git fetch
+    git reset --hard origin/$branch
```

Comment:
maybe ask for confirmation if there's local changes? this may destruct some users' working states and surprise them